scipy/constants/_codata.py

```diff
@@ -1852,40 +1985,82 @@
 Wien wavelength displacement law constant                   2.897 771 955... e-3     (exact)                  m K
 W to Z mass ratio                                           0.881 45                 0.000 13                    """
 
-# -----------------------------------------------------------------------------
 
-physical_constants: dict[str, tuple[float, str, float]] = {}
+exact2022 = exact2018
```

Comment:
This was initially a guess, but I think it is correct - all of the values that were exact in 2018 are exact in 2022, there are no new exact values in 2022, and nothing about the values has changed (so I assume they can still be computed the same way).